bin/spring

```diff
@@ -46,4 +46,5 @@ end
 lib = File.expand_path("../../lib", __FILE__)
 $LOAD_PATH.unshift lib unless $LOAD_PATH.include?(lib) # enable local development
 require 'spring/client'
+Spring.verbose_output = false if $0.include? 'spring' # don't output running spring status if command has `spring` in it
```

Comment:
ah yeah, makes sense

On Sat, Dec 12, 2015 at 11:44 PM, Aditya Prakash <notifications@github.com>
wrote:

> In bin/spring
> <https://github.com/rails/spring/pull/455#discussion_r47441438>:
>
> > @@ -46,4 +46,5 @@ end
> >  lib = File.expand_path("../../lib", __FILE__)
> >  $LOAD_PATH.unshift lib unless $LOAD_PATH.include?(lib) # enable local development
> >  require 'spring/client'
> > +Spring.verbose_output = false if $0.include? 'spring' # don't output running spring status if command has `spring` in it
>
> how about: Spring.verbose_output = $0.include?('spring')
>
> I think you meant Spring.verbose_output = !$0.include?('spring')
> It is not going to work it will set Spring.verbose_output to true when
> command includesspring, even when user has Spring.verbose_output = false
> defined in config/spring.rb (or ~/.spring.rb).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/spring/pull/455/files#r47441438>.
>